README.md

```diff
@@ -1,2 +1,57 @@
+![GitHub forks](https://img.shields.io/github/forks/beyond-io/picATrip?style=social) ![GitHub issues](https://img.shields.io/github/issues-raw/beyond-io/picATrip?logo=GitHub) ![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/beyond-io/picATrip?logo=GitHub) ![GitHub top language](https://img.shields.io/github/languages/top/beyond-io/picATrip)
 # picATrip
-picATrip - Beyond 05, team 2 project
+![logo- picATrip](https://user-images.githubusercontent.com/48455448/110465508-26ae0a80-80dd-11eb-8268-586b26a1e59e.png)
+
+### **Who are we?**
+
+   picATrip is A web platform for **sharing experiences from _trips_** in a form of **photos** based
+   on various parameters as
+   location/duration time/difficulty and provides a trip **search functionality** and **commenting
+   functionality**.
+
+   It allows users to have the opportunity to **contribute their trip experience** and thoughts by
+   **pic**king  a picture of a **trip** shared on the platform,  and by that **learn about the trips**
+   across the country that stands behind the pictures and having the opportunity to **share their
+   thoughts** and **comment** if they desire.
+
+### Why using us?
+  **Ever heard of the phrase 'eating with the eyes'?**, well that's about it! we believe in outdoor lifestyle and trips, and we strive
+  to promote and encourage people to share their experience of trips in a form of a photo, only then when a user had an interest
+  in a trip picture, he would be capable of getting more information about the trip, and comment his thoughts about it.
+
+  On days where we spending most of our time indoors, staring at screens, and being glued to our chair, whether it's as part of
+  our studies, work, or as part of binging a series on the couch.
+  We desire to change it and as wiser people have said- a photo worth a thousand words,
+  then what's better than a thousand words to describe an experience of a trip.
+
+### still not convinced?
+ as my mom used to say to me as a picky eater- _" try first, and then we'll discuss it, what do you have to lose?"_
+
+
+## Tools used for Developmenta
```

Comment:
I think that you added an 'a' in the end of the sentence by mistake.

Reply:
thanks, got fixed in the last version